### ISSUE TITLE
VxScanners: export relevant settings with CVRs

### ISSUE THREAD

**Issue by eventualbuddha** [votingworks/vxsuite#1606]
## Summary
We have no record of the thresholds used to determine whether marks should be counted as votes in the CVR export. As the values can be adjusted at any time, we need to add the thresholds used with every CVR entry. If there are any other relevant settings, we should add those too.

## Requirements
CVR entries should contain the effective `markThresholds` values used for that sheet.

## Non-Requirements
We don't need to change anything about the way scanning or threshold overrides work, just write the values to the CVR.

## Tasks
- [ ] Record with each sheet what the thresholds used were.
- [ ] Write that information to the CVR export for each entry, probably as `_markThresholds`.

**Comment by carolinemodic** [comment]
We should tackle this when we reevaluate the CVR export format for VVSG.

**Comment by adghayes** [comment]
What does having the mark thresholds in the CVR allow you to later do in VxAdmin? Is this purely for accounting purposes?

Say the `markThreshold` is 0.12. We have CVRs with that `markThreshold: 0.12`. Later, we adjust the `markThreshold` to 0.15. We don't know how to re-interpret the marks from `markThreshold: 0.12` unless we have the individual mark values for each mark. 

The case is worse if we want to lower the mark threshold, in which case we haven't recorded marks which may not be counted.

**Comment by adghayes** [comment]
Talked with @carolinemodic and confirmed we still want to do this. Even though we are including the mark thresholds for all marks, it still provides a helpful piece of information to understand how we got from the "original" to the "modified" snapshot.

**Comment by carolinemodic** [comment]
@adghayes @mattroe Do we want this to be for mvcp?

**Comment by adghayes** [comment]
I don't think it needs to be given that we're including mark thresholds for all bubbles.
